doc/modules/grid_search.rst

```diff
@@ -1,10 +1,10 @@
-.. currentmodule:: sklearn.grid_search
+.. currentmodule:: sklearn.model_selection
 
 .. _grid_search:
 
-===============================================
-Grid Search: Searching for estimator parameters
-===============================================
+==========================================
+Search: Searching for estimator parameters
+==========================================
 
 Parameters that are not directly learnt within estimators can be set by
 searching a parameter space for the best :ref:`cross_validation` score.
```

Comment:
I would give the ``cross_validation`` here an explicit name. Currently it looks really awkward.